SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -123,7 +123,7 @@ class CSourceBase_TurbSA : public CNumerics {
  * \class CSourcePieceWise_TurbSA
  * \brief Class for integrating the source terms of the Spalart-Allmaras turbulence model equation.
  * \ingroup SourceDiscr
- * \author A. Bueno.
+ * \author G. Suarez
```

Comment:
That's a bit unfair...

Reply:
I wrote it to put on record who has modified the original version, just in case in a future someone finds any issue with that implementation. The same for CSourcePieceWise_TurbSA_Neg.

Reply:
I got the idea, we don't have guidelines for this, but I only replace authors if I re-write the entire thing, when I make significant changes I add myself to the list, and for fixing bugs I just let git do the tracking of who modified which line.